mungegithub/mungers/blunderbuss.go

```diff
@@ -125,11 +126,25 @@ func getPotentialOwners(author string, feats *features.Features, files []*github
 			if owner == author {
 				continue
 			}
+			// Inside of a vendor directory, it is possible that
+			// we will encounter OWNERS entries that are not valid
+			// assignees in the repository we are munging. We need
+			// to ignore those owners.
+			if strings.HasPrefix(file.GetFilename(), "vendor") {
+				ok, err := obj.IsAssignee(owner)
+				if err != nil {
+					return nil, 0, err
```

Comment:
I expect this to be optional extra information, should it fail the function altogether if we can't know?